include/swift/SIL/SILType.h

```diff
@@ -228,26 +228,29 @@ class SILType {
   /// something of unknown size.
   ///
   /// This is equivalent to, but possibly faster than, calling
-  /// M.Types.getTypeLowering(type).isAddressOnly().
-  static bool isAddressOnly(CanType T, SILModule &M,
-                            CanGenericSignature Sig,
-                            ResilienceExpansion Expansion);
+  /// tc.getTypeLowering(type).isAddressOnly().
+  static bool isAddressOnly(CanType type, Lowering::TypeConverter &tc,
```

Comment:
changing the API from SILModule to TypeConverter is not strictly required, right? isAddressOnly could still get the type converter reference from the module.

Reply:
ok, makes sense